feature/addbook/src/main/java/com/sopt/anshim/addbook/type/AddBookEvent.kt

```diff
@@ -0,0 +1,33 @@
+package com.sopt.anshim.addbook.type
+
+import android.net.Uri
+
+/**
+ * AddBookScreen의 사용자 이벤트
+ *
+ * [TitleChanged] 는 도서 제목을 변경하는 이벤트
+ * [AuthorChanged] 는 저자를 변경하는 이벤트
+ * [PriceChanged] 는 가격을 변경하는 이벤트
```

Comment:
```suggestion
/**
 * AddBookScreen의 사용자 이벤트
 *
 * @property TitleChanged 도서 제목을 변경하는 이벤트
 * @property AuthorChanged 저자를 변경하는 이벤트
```

요런식으로 써줘도 좋을 거 같아요~~!!